src/tbots.py

```diff
@@ -118,16 +118,25 @@
             ]
```

Comment:
While testing the new ./tbots.py script, I noticed that it adds an empty string target as well, which sometimes gets picked as the best target. This is because the result of the `run` query ends with a new line. We should update line 116 to `run(query, stdout=PIPE).stdout.rstrip(b"\n").split(b"\n")`.

Reply:
changed.